### PR TITLE
Mass refactoring and AddRecipeCommand

### LINKED ISSUE #6: As a new user, I can add new recipes on the computer
so that I can create recipes on the computer

### PR DESCRIPTION
**Pull request by daongochieu2810**

### CODE REVIEW

**Review by oliviajohansen — APPROVED** [review]
thanks lgtm

**Review by jiaqi20 — APPROVED** [review]
lgtm!!!

**Review by caitlinjee — APPROVED** [review]
Lgtm

File: README.md [hunk 1]
```diff
@@ -15,4 +15,15 @@ For the detailed documentation of this project, see the **[Wishful Shrinking Pro
...
   
+  
+* This is **a sample project for Software Engineering (SE) students** .<br>
+  Example usages:
+  * as a starting point of a course project (as opposed to writing everything from scratch)
+  * as a case study
+* The project simulates an ongoing software project for a desktop application (called _WishfulShrinking_) used for managing contact details.
+  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
+  * It comes with a **reasonable level of user and developer documentation**.
+* It is named `WishfulShrinking Level 3` (`AB3` for short) because it was initially created as a part of a series of `WishfulShrinking` projects (`Level 1`, `Level 2`, `Level 3` ...).
```
Comment (TianYong-Goh) [inline README.md:26]: Would it be better to remove this statement? As I feel our product is not a series of the addressbook.

File: docs/team/johndoe.md [hunk 2]
```diff
@@ -3,9 +3,9 @@ layout: page
 title: John Doe's Project Portfolio Page
 ---
 
-## Project: AddressBook Level 3
+## Project: WishfulShrinking Level 3
```
Comment (TianYong-Goh) [inline docs/team/johndoe.md:6]: Is it better to remove these Level 3?

**Review by TianYong-Goh — APPROVED** [review]
lgtm. just a little minor change you might want to consider. Thank you.

**Pull request merged**